api/descriptions/feeds/ips_digest_get.md

```diff
@@ -1,11 +1,18 @@
-Retrieves the digest from the list of suspicious or high risk IP endpoints configured in the console.
+Retrieves the digest string for the list of suspicious or high risk IP endpoints configured in the console.
```

Comment:
@KICKdesigns For our two main components (Console and Defender), please capitalize throughout. For style, we refer to them as "Console" and "Defender", not "the Console" or "the Defender".

Please rephrase as:

`Retrieves the digest string for the list of suspicious or high risk IP endpoints configured in Console.`